test/ui-test/src/common/Common.py

```diff
@@ -0,0 +1,12 @@
+
+
+class Common:
```

Comment:
I'm not sure about this `Common` class. Where will it be used? Now as I can see it is a wrapper of the wrapper (double wrapper :face_with_spiral_eyes:) we have of the squish driver... why not using directly the `SquishDriver.py`?

Reply:
It's not in use yet. Once it is, I will explain further.